src/Mount.cpp

```diff
@@ -2332,6 +2311,21 @@ float Mount::getSpeed(int direction) {
   return 0;
 }
 
+/////////////////////////////////
+//
+// calculatePositions
+//
+// This code calculates the stepper locations to move to, given the right ascension and declination
+/////////////////////////////////
+void Mount::calculatePositions(float raCoord, float decCoord, long& raPos, long& decPos){
```

Comment:
I would rather name it `calculateTargetPositions` and the param names accordingly. Otherwise it can be misunderstood as "current position"

Reply:
Well.... it doesn't calculate the target position of the mount, just as it doesn't calculate the current position. How about calculateStepperPositions?

Reply:
Fixed